### ISSUE TITLE
elastic4s-http: Exception on generating dateHistogramAggregations

### ISSUE THREAD

**Issue by halvorgb** [sksamuel/elastic4s#944]
`com.fasterxml.jackson.core.JsonGenerationException: Can not write a field name, expecting a value` is thrown when using DateHistogramAggregation.extendedBounds

using elastic4s-http version 5.4.3

reproduction:
```scala
import com.sksamuel.elastic4s.http.ElasticDsl._
import com.sksamuel.elastic4s.http.HttpClient
import com.sksamuel.elastic4s.ElasticsearchClientUri
import org.elasticsearch.search.aggregations.bucket.histogram.ExtendedBounds

val client = HttpClient(ElasticsearchClientUri("localhost", 9200))

val qStrings = search("index" / "type").aggs{
  dateHistogramAggregation("histogram")
    .field("timestamp")
    .extendedBounds(new ExtendedBounds("2017-01-01", "2018-01-01"))
}
  .size(0)

val qLongs = search("index" / "type").aggs{
  dateHistogramAggregation("histogram")
    .field("timestamp")
    .extendedBounds(new ExtendedBounds(0, 100))
  }
  .size(0)

// these two will fail with the JsonGenerationException
client.show(qStrings)
client.show(qLongs)

```

**Comment by halvorgb** [comment]
This issue seems to be fixed in 5.4.7, closing.

**Comment by halvorgb** [comment]
Thanks :)
